_posts/2015-06-12-apac-day-of-coderetreat-on-jun-13.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+author: JosephYao
+authorname: Joseph Yao
+title: APAC Day of Coderetreat on Jun 13
+preview: 'As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture.'
+---
+
+
+As you may know, software craftsman and coderetreat culture is much weaker in APAC (Especially in China) than US and Europe. So, I got an idea to organize a day event of coderetreat (like GDCR) to promote that culture. After I talked to other GDCR organisers in China and Asia, the idea was accepted quite well. Then, the date was set at Jun 13 (Saturday).
+
+
+We now have 14 cities registered for this event and I created a map  [here](https://www.google.com/maps/d/viewer?mid=zFRRheM6-9AQ.k8wlhsC1wCNY&amp;usp=sharing).  
+
+
+![embedded map](https://www.google.com/maps/d/u/0/embed?mid=1lsAFrsPkiHnp0vUpqW_50NJQiuI)
+
+
+P.S. The intention is to have a light way regional coderetreat event in APAC. So, there will be less material and activities required comparing to GDCR. E.g. there is no plan to have sponsor for the whole event, no printout, no slides, etc. Every location will organise the event on their own and we’re on the same day. And, this event is an addition to GDCR, not a replacement.
+
+
+P.P.S. Here is an [article link](http://blog.csdn.net/uxyheaven/article/details/46488069) (in Chinese) and a picture of article posted in a company's internal magazine.
+
+
+![APAC day of coderetreat]({site.url}/images/APACdayofCoderetreat.jpg)
```

Comment:
I don't think `site.url` is needed here. Jekylls [`link`](http://jekyllrb.com/docs/templates/#link) will make sure resources are valid when jekyll compiles the page, so I'd rather use that instead.